apps/backend/src/app/categories/categories.service.ts

```diff
@@ -0,0 +1,49 @@
+import { Injectable } from '@nestjs/common';
+import { InjectRepository } from '@nestjs/typeorm';
+import { EntityNotFoundError, Repository } from 'typeorm';
+import { CreateCategoryDto } from './dto/create-category.dto';
+import { UpdateCategoryDto } from './dto/update-category.dto';
+import { Category } from './entities/category.entity';
+
+@Injectable()
+export class CategoriesService {
+  constructor(
+    @InjectRepository(Category)
+    private readonly categoriesRepository: Repository<Category>
+  ) {}
+
+  create(createCategoryDto: CreateCategoryDto) {
+    const category = this.categoriesRepository.create(createCategoryDto);
+
+    return this.categoriesRepository.save(category);
+  }
+
+  findAll() {
+    return this.categoriesRepository.find();
+  }
+
+  async findOne(id: number) {
+    const category = await this.categoriesRepository.findOneOrFail({ where: { id } });
+
+    return category;
```

Comment:
You can simplify and write this method in single line

```typescript
async findOne(id: number) {
  return this.categoriesRepository.findOneOrFail({ where: { id } });
}
```

Reply:
RIght, I had something else between these lines before and forgot to clean up :)